### DISCUSSION TITLE
地球拼音中，如何设置问号为轻声？

### DISCUSSION THREAD

**Discussion by SoloGiant** [rime/home#794] [Q&A]
之前改过分号为轻声，
   将    
 - 'xlit 1234 -/<\'     修改为    
 - 'xlit 12345 -/<\;'
注释掉    - 'derive/^(.*)-$/$1;/'  
   将  
- 'xform a[-;] ā'     修改为     
- 'xform a[-] ā'
-
 添加   - 'xform a[;] a'等，
之后就可以分号作为轻声了。

现在想使用问号作为轻声，将上述修改中的分号改为问号后，添加binding：
    - { when: has_menu, accept: question, send: question } 

不成功，依旧问号直接上屏。
请问还要做哪些修改？

**Comment by lotem** [comment]
聲調符號須採用 `speller/alphabet` 中的字符。

**Reply by SoloGiant** [reply]
已经将“？”添加到speller/alphabet了，依旧不行。字典中已经将轻声改为“pinyin5”形式，当前配置如下
speller:
    alphabet: 'zyxwvutsrqponmlkjihgfedcba-;/<,>\?'
    algebra:
     ...
      - 'xlit 12345 -/<\?'
translator:  
     preedit_format:
      ...
      - 'xform a[?] a'
      - 'xform e[?] e'

**Reply by SoloGiant** [reply]
> 聲調符號須採用 `speller/alphabet` 中的字符。

怀疑是bug，同样配置，W10无法使用问号输出轻声，Win7（虚拟机）可以。
